packages/flutter_tools/bin/fuchsia_tester.dart

```diff
@@ -85,6 +87,7 @@ Future<Null> run(List<String> args) async {
     if (!fs.isDirectorySync(sdkRootSrc.path)) {
       throwToolExit('Cannot find SDK files at ${sdkRootSrc.path}');
     }
+    final String coverageDirectory = argResults[_kOptionCoverageDirectory];
```

Comment:
Seems like this should be held and passed around as a `Directory` rather than a `String` (`fs.directory(argResults[_kOptionCoverageDirectory])`)

Reply:
Done